config/cfg.py

```diff
@@ -0,0 +1,18 @@
+from cmath import pi
+
+N = 513
+
+Fs = 1e6
+Ts = 1.0 / Fs
+
+A = 1
+phi = pi / 8.0
+sigma_noise = 1.0
```

Comment:
Shouldn't sigma_noise be calculated from SNR? We could therefore just have SNR in the config-file

Reply:
sigma_noise is used in https://github.com/chrstrom/monkey_estimation/blob/b4bb576b833047172930aee5e73e14c23b74e940/scripts/Signals/Signals.py#L36 to generate the w[n] in x[n]. I just renamed it from "sigma" to "sigma_noise"

Reply:
Yes, but this value is not known beforehand. We only know it from the SNR

Reply:
Gotcha, fixed!